neuromation/cli/docker_credential_helper.py

```diff
@@ -18,19 +18,29 @@ def error(message: str, exit_code: int) -> None:
     elif action == "erase":
         print("Please use `neuro logout` instead `docker logout ...`", EX_UNAVAILABLE)
     else:
-        async with get() as client:
-            config = client.config
-            registry = sys.stdin.readline().strip()
-            neuro_registry = config.registry_url.host
-            if registry != neuro_registry:
-                error(
-                    f"Unknown registry {registry}. "
-                    "neuro configured with {neuro_registry}.",
-                    EX_DATAERR,
-                )
-            token = await config.token()
-            payload = {"Username": "token", "Secret": token}
-            print(dumps(payload))
+        registry = sys.stdin.readline().strip()
+        try:
+            async with get() as client:
+                config = client.config
+                neuro_registry = config.registry_url.host
+                if registry != neuro_registry:
```

Comment:
seems like here we need to check against 
https://github.com/neuromation/platform-client-python/blob/b1b5df046c0c162f305435180f21fdd38a38fba1/neuromation/api/config.py#L89-L91

Reply:
here 
https://github.com/neuromation/platform-client-python/blob/b1b5df046c0c162f305435180f21fdd38a38fba1/neuromation/cli/config.py#L193-L194
we are adding a single cluster, although a user can have multiple clusters.